Zend/zend_portability.h

```diff
@@ -135,16 +135,26 @@
 #  define RTLD_LAZY 1    /* Solaris 1, FreeBSD's (2.1.7.1 and older) */
 # endif
 
+# ifndef RTLD_NOW
+#  define RTLD_NOW  2
```

Comment:
This define seems dubious, I'd prefer a compile error if it doesn't exist.

Reply:
It was a copy paste from above fir RTLD_LAZY ;)
BTW, I will remove it

Reply:
Notice than, from libtool, some platform may use DL_LAZY, DL_NOW

```
#ifndef LT_DLLAZY_OR_NOW
#  ifdef RTLD_LAZY
#    define LT_DLLAZY_OR_NOW		RTLD_LAZY
#  else
#    ifdef DL_LAZY
#      define LT_DLLAZY_OR_NOW		DL_LAZY
#    else
#      ifdef RTLD_NOW
#        define LT_DLLAZY_OR_NOW	RTLD_NOW
#      else
#        ifdef DL_NOW
#          define LT_DLLAZY_OR_NOW	DL_NOW
#        else
#          define LT_DLLAZY_OR_NOW	0
#        endif
#      endif
#    endif
#  endif
#endif
```